satpy/tests/reader_tests/test_satpy_cf_nc.py

```diff
@@ -64,12 +64,14 @@ def setUp(self):
                            dims=('y', 'x'),
                            coords={'y': y_visir, 'x': x_visir},
                            attrs={'name': 'lat',
-                                  'standard_name': 'latitude'})
+                                  'standard_name': 'latitude',
+                                  'modifiers': np.array([])})
```

Comment:
Does this make the code fail if your change isn't implemented?

Reply:
Yes, exactly